contracts/governance/compatibility/GovernorCompatibilityBravo.sol

```diff
@@ -39,6 +39,11 @@ abstract contract GovernorCompatibilityBravo is IGovernorTimelock, IGovernorComp
 
     mapping(uint256 => ProposalDetails) private _proposalDetails;
 
+    /**
+     * @dev Vote type is not in the {VoteType} enum.
+     */
+    error GovernorInvalidVoteType();
```

Comment:
This one can be expected to show up in any counting module, we should define it in `IGovernor`.

Reply:
Moved with the following understanding:

- `GovernorInvalidVoteType` is a generic `Governor` error because a counting module is required and this may throw in that module
- `VoteType` is defined in each counting module.